index.php

```diff
@@ -1,6 +1,11 @@
 <?php 
-require_once('functions.php');
-
+require_once('CollectionCardMaker.php');
```

Comment:
for the unit testing task you will need to put this in a src folder and then update all of the requires to match this